### ISSUE TITLE
Gallery page

### ISSUE THREAD

**Issue by danbri** [notlion/YASE#4]
Pity there is no gallery page at http://yase.chnk.us/ ... is the project alive anywhere, or a successor?

**Comment by notlion** [comment]
Agreed.. It really needs a gallery. Even I have a hard time finding the good shaders people have made. There is a WIP [rewrite](https://github.com/notlion/pst) that supports colors, but that has been stale for a while. Perhaps I'll have a minute to revisit this next month.

**Comment by brucelane** [comment]
yes please, YASE sounds great!

**Comment by danbri** [comment]
Thanks - hope you find time! 

I stumbled across a couple of incoming links to different examples but they're really hard to search for. Even a few nice extra example entries linked from this bug would be super-handy.

 https://lotsacode.wordpress.com/2013/04/16/fun-with-particles-yet-another-shader-editor/ has a few examples. 

Also found by guessing or links http://yase.chnk.us/#d2 (my favourite) http://yase.chnk.us/#f1http://yase.chnk.us/#g1 http://yase.chnk.us/#m1 http://yase.chnk.us/#v1 http://yase.chnk.us/#s1 http://yase.chnk.us/#w1 
http://yase.chnk.us/#2 http://yase.chnk.us/#3 http://yase.chnk.us/#4 http://yase.chnk.us/#5 (my other favourite...), http://yase.chnk.us/#6 http://yase.chnk.us/#7 http://yase.chnk.us/#12 (moody...), http://yase.chnk.us/#14  http://yase.chnk.us/#15   ... etc.

**Comment by notlion** [comment]
Here's a few that I've saved:

http://yase.onecm.com/#ts
http://yase.onecm.com/#lm

http://yase.onecm.com/#w2
http://yase.onecm.com/#27
http://yase.onecm.com/#36
http://yase.onecm.com/#v0

Orbits
http://yase.onecm.com/#vt
http://yase.onecm.com/#sk
http://yase.onecm.com/#vf
http://yase.onecm.com/#vs
http://yase.onecm.com/#scx
http://yase.onecm.com/#86

Cubes
http://yase.onecm.com/#g5
http://yase.onecm.com/#p2
http://yase.onecm.com/#7v
http://yase.onecm.com/#2f
http://yase.onecm.com/#jz
http://yase.onecm.com/#gzy

Snow spheres & cubes
http://yase.onecm.com/#l0
http://yase.onecm.com/#zt

Menger
http://yase.onecm.com/#wh

Planet
http://yase.onecm.com/#db
http://yase.onecm.com/#gj
http://yase.onecm.com/#14
http://yase.onecm.com/#ys

http://yase.onecm.com/#x4

IFS
http://yase.onecm.com/#18

**Comment by danbri** [comment]
another entry point - http://slides.com/notlion/pst#/7
